### DISCUSSION TITLE
Grupo de exemplo com bot ativo 🤖

### DISCUSSION THREAD

**Discussion by elizandrodantas** [elizandrodantas/bot-blaze-telegram#12] [Announcements]
## grupo do telegram
exemplo de funcionamento do codigo [24hrs]: link  **https://t.me/+w6nZvuahQvY0ZTZh**

**Comment by DinhoNunesLC** [comment]
Aqui uma sala 24h para Double: https://t.me/sinaisbdouble
Aqui uma sala 24h para Crash: https://t.me/sinaisbcrash
